test/any/datastore/rate_limited_DAO-test.es6.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('RateLimitedDAO', function() {
+  let rateLimitedDAO;
+  foam.CLASS({
```

Comment:
Required foam2/test/helpers/testcontext.js in webpack-helper.js. Bumped foam.CLASS to beforeEach.